design/006-LTS-release.md

```diff
@@ -0,0 +1,50 @@
+# Long Term Support Policy
+
+WRT: https://github.com/external-secrets/external-secrets/issues/2044
+
+We want to provide security patches and critical bug fixes in a timely manner to our users.
+To do so, we offer long-term support for our latest two (N, N-1) software releases.
+We aim for a 2-3 month **minor** release cycle, i.e. a given release is supported for about 4-6 months.
+
+We want to cover the following cases:
+
+* weekly image rebuilds to update OS dependencies
+* weekly go dependency updates
```

Comment:
dependency updates **must be approved** and merged by a maintainer. I.e. this is not fully automatic.